### PR TITLE
Add GitHub Actions runner chart

### PR DESCRIPTION
**Pull request by damacus**
- Adds Helm Chart for GitHub Actions runner
- Adds example values.yaml
- Adds a deployment for the single container pod
- Adds a secret for the GitHub Organisation token

### CODE REVIEW

File: charts/github-actions-runner/values.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+image:
+  repository: myoung34/github-runner
```
Comment (andylightfoot) [inline charts/github-actions-runner/values.yaml:3]: I'm not sure I'm that keen on us using docker images from personal repos for our github runners

File: charts/github-actions-runner/values.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+env:
+  org:
+    runner: "true"
+    name: ""
+    secret:
+      access_token:
+        name: github-token
+  labels: "kubernetes"
+
+autoscaling:
```
Comment (andylightfoot) [inline charts/github-actions-runner/values.yaml:16]: How does the autoscaling work?

File: charts/github-actions-runner/values.yaml (hunk 1 above)
Comment (damacus) [reply charts/github-actions-runner/values.yaml:3]: Granted. Though, this looks to be the best community supports Docker image. 

We could look at cloning and building our own, but that seems like a lot of extra work for nebulous gains.

File: charts/github-actions-runner/values.yaml (hunk 2 above)
Comment (damacus) [reply charts/github-actions-runner/values.yaml:16]: I'm fairly sure it works with CPU "pressure". Happy to move to a fixed of runners if this doesn't work in heavy testing.

File: charts/github-actions-runner/values.yaml (hunk 1 above)
Comment (andylightfoot) [reply charts/github-actions-runner/values.yaml:3]: These runners will have potentially very powerful IAM permissions in our AWS account so I think it is wrong to discuss concerns about the security of this image as nebulous.

File: charts/github-actions-runner/values.yaml (hunk 2 above)
Comment (andylightfoot) [reply charts/github-actions-runner/values.yaml:16]: Yeah but what is actually scaling it based on CPU Pressure? I can't see it adding a Horizontal Pod Autoscaler

File: charts/github-actions-runner/values.yaml (hunk 1 above)
Comment (andylightfoot) [reply charts/github-actions-runner/values.yaml:3]: I don't think this should block the POC of github runners but lets review and come to a consensus on this before giving too many IAM perms

**Review by andylightfoot — APPROVED** [review]
Maybe go with fixed number of replicas for now then work out autoscaling later on?

**Pull request merged**